src/User32.Desktop/User32.cs

```diff
@@ -1626,6 +2151,14 @@ public static partial class User32
         [DllImport(nameof(User32), SetLastError = true)]
         public static extern void PostQuitMessage(int nExitCode);
 
+        [DllImport(nameof(User32), SetLastError = true)]
+        [return: MarshalAs(UnmanagedType.Bool)]
+        public static extern bool PostThreadMessage(uint threadId, PInvoke.User32.WindowMessage msg, UIntPtr wParam, IntPtr lParam);
```

Comment:
We already define this method. We have a policy of avoiding overloads of such multi-purpose methods. But if you want to change the existing method to use `void*` for its last two parameter types instead of `IntPtr`, the code generator will recreate the `IntPtr` overload.